src/components/deletePostModal/DeletePostModal.tsx

```diff
@@ -11,25 +25,65 @@ type DeletePostModalType = {
   absoluteRef: RefObject<HTMLDivElement>
   handleCloseDelete: () => void
   handleDeletePost: () => void
+  handleInputChange: (event: any) => void
+  handleIsDisabled: boolean
 }
 
 const DeletePostModal: FC<DeletePostModalType> = ({
   decodedTitle,
   overlayRef,
   absoluteRef,
   handleDeletePost,
-  handleCloseDelete
+  handleCloseDelete,
+  handleInputChange,
+  handleIsDisabled
 }) => {
+  
   return (
     <DeletePostModalStyled>
       <Overlay
         ref={overlayRef}
         onClick={handleCloseDelete} />
       <AbsoluteDeleteWrapper ref={absoluteRef}>
-        <h1>{ decodedTitle }</h1>
-        <button onClick={handleDeletePost}>
-          Delete
-        </button>
+          <CloseModalButtonWrapper
+            onClick={handleCloseDelete} >
+            <CloseModalButton color="white"/>
+          </CloseModalButtonWrapper>
+          <DeleteModalHeader>
+            <h2 className="title">Delete Poem</h2>
+          </DeleteModalHeader>
+          <AlertModalWrapper>
+            <AlertModalMessage>
+              <p>You cannot recover this poem once it is deleted!</p>
+            <AlertModalTriangleWrapper>
+              <AlertModalTriangle color='rgba(200, 139, 28, 1)' />
+            </AlertModalTriangleWrapper>
+            </AlertModalMessage>
+          </AlertModalWrapper>
+        <ConfirmForm>
+          <p>This will delete your poem:</p>
+          <h2 className="title">{ decodedTitle }</h2>
+          <label htmlFor='confirm'>
+            <p>Type name of poem to confirm.</p>
+          </label>
+          <InputWrapper>
+            <Input 
+              type='text' 
+              id='confirm' 
+              name='confirm' 
+              placeholder="Type in name of poem"
+              onChange={handleInputChange}
+              />
+              <AlertModalCircleWrapper>
+                <AlertModalCircle color='rgba(199, 81, 86, 1)'/>
+              </AlertModalCircleWrapper>
+          </InputWrapper>
+          <DeleteButton className='hover'
+            disabled={handleIsDisabled}
```

Comment:
Typically, we tend to reserve `handle` for event listener type things. In this case, `handleIsDisabled` is representing a boolean value. Perhaps something along the lines of `isDisabled` may be more descriptive.